profiles/common/modules/features/nexteuropa_newsroom/nexteuropa_newsroom.blocks.inc

```diff
@@ -0,0 +1,889 @@
+<?php
+
+/**
+ * @file
+ * To change this license header, choose License Headers in Project Properties.
+ *
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+
+/**
+ * Implements hook_block_info().
+ */
+function nexteuropa_newsroom_block_info() {
```

Comment:
make them cached at least otherwise these blocks will generate long page load and performance issues based on what are implemented there.